### DISCUSSION TITLE
Book Format

### DISCUSSION THREAD

**Discussion by TWebSrDev** [chrisminnick/react-js-foundations#3] [Q&A] [answered]
Hi Chris!

Man, based on the listed content, this looks like one of the best React books written thus far. The only other one in which I immediately wanted to -and did- purchase as soon as it was released, was written by the great Adam Freeman (I pretty much buy all of his books on topics-of-interest on reflex). That said, you can't imagine my disappointment when I discovered it is only available electronically in proprietary format (VitalSource and Kindle), requiring a separate e-reader. It is because of this reason that I don't buy e-books from Amazon, and buy the vast majority of my (100's of) tech-books from SpringerLink (previously Apress) and Manning Publications. 

So my question is, do you foresee your book being available in standard PDF format at some point?

Thanks,
Tony

**Comment by chrisminnick** [comment, marked as answer]
Hi Tony,

Thanks for writing! I'm really proud of the book, and I think it's pretty good too. 

I asked the publisher (Wiley) a couple months ago for a PDF and they said they didn't do that. I just sent another email asking whether they plan to in the future, and I'll let you know what I find out.

Best,
-Chris

**Reply by TWebSrDev** [reply]
Hey Chris, thanks so much for your quick response...Much appreciated! 

I really hope that you can work something out with them. One possible solution might be that the book could still be purchased from Wiley, but instead of downloading from VitalSource, they could provide the user with a code, and they could maybe email you for a PDF. I mean, the books are DRM'ed, so not sure what their issue is. That said, I actually wouldn't MIND a proprietary format if all the publishers could get together an agree on ONE format and reader that allows the user to actually "own" their copy of the book. The problem with Amazon, is they're only selling ACCESS to your content, which they can then arbitrarily deny, for any or no reason at all...Which I find completely absurd.

Again, thanks!
Tony
